django/cantusdb_project/main_app/forms.py

```diff
@@ -879,3 +880,17 @@ class Meta:
     complete_inventory = forms.ChoiceField(
         choices=TRUE_FALSE_CHOICES_INVEN, required=False
     )
+
+
+class AdminUserChangeForm(forms.ModelForm):
+    class Meta:
+        model = get_user_model()
+        fields = "__all__"
+
+    email = forms.CharField(
+        required=True,
+        widget=AdminTextInputWidget,
+    )
+    email.widget.attrs.update({"style": "width: 300px;"})
+
+    password = ReadOnlyPasswordHashField()
```

Comment:
what does this do?

Reply:
When I initially set up the user admin change form, the user's passwords would be displayed as text (hashed, but editable and unhidden). Using the ReadOnlyPasswordHashField maintains the use of this widget instead. However, I notice now that it is missing the same help text that takes the user to the password change form, so I added it here as well.